rtengine/imageio.h

```diff
@@ -90,6 +90,10 @@ class ImageIO : virtual public ImageDatas
     int load (const Glib::ustring &fname);
     int save (const Glib::ustring &fname) const;
 
+#ifdef LIBJXL
+    int loadJxl (const Glib::ustring &fname);
```

Comment:
Hmm, the other `load...()` extensions are uppercase. And better append the new one rather than prepend it.